src/Components/Slider.js

```diff
@@ -112,6 +117,7 @@ class CustomizedSlider extends React.Component {
         <Nouislider
           start={value}
           key={sliderKey}
+          connect
```

Comment:
@lambert3 , regarding this question https://github.com/Geo-Gecko/ASRT_GG/pull/11#discussion_r454905786 . I was asking about this connect function you're passing into nouislider

Reply:
nouislider has the connect option to control the slider bar. For instance the color of the bar depends on the connect option.